server/src/main/java/io/crate/analyze/AnalyzedTableElements.java

```diff
@@ -504,14 +512,16 @@ public static <T> void buildReference(RelationName relationName,
                 (Integer) geoMap.get("tree_levels"),
                 distError != null ? distError.doubleValue() : null
             );
-        } else if (bound && columnDefinition.analyzer() != null) {
+        } else if (bound && (columnDefinition.analyzer() != null || columnDefinition.copyTo().isEmpty() == false)) {
+            // If analyzer is not null, it's a column definition with inlined FULLTEXT.
+            // If columnDefinition.copyTo() is not empty, it's a column definition referred by a table level INDEX declaration.
             // We need to use "all fields" constructor to make sure we cover all possible options when used in CREATE TABLE
             ref = new IndexReference(
                 new ReferenceIdent(relationName, columnDefinition.ident()),
                 RowGranularity.DOC,
                 DataTypes.STRING,
                 ColumnPolicy.STRICT, // Use relevant for non-object field value.
```

Comment:
not sure STRICT is relevant, any non-null values is fine since it's not relevant anyway and needed only to prevent NPE on streaming. 

Could pass null and adjust streaming to make it optional - just thought that it's unnecessary complexity and passing dummy is easier?